src/IdentityManager/WebApi/Models/RegisterViewModel.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace IdentityManager.API.Models
+{
+    public class RegisterViewModel
```

Comment:
I suppose this model is used, when new Agency is registered (with new user itself). 
Please check with mocks, which fields we require. There should be much more fields.